test/helpers/hosts_and_hostgroups_helper_test.rb

```diff
@@ -187,9 +187,24 @@ def setup
     @hostgroup.content_source = @content_source
 
     @host = ::Host.new(:architecture => @arch, :operatingsystem => @os,
-                      :content_facet_attributes => {:lifecycle_environment_id => @env.id,
-                                                    :content_view_id => @cv.id,
-                                                    :content_source_id => @content_source.id})
+                       :content_facet_attributes => {:lifecycle_environment_id => @env.id,
+                                                     :content_view_id => @cv.id,
+                                                     :content_source_id => @content_source.id},
+                       :location => Location.new(
+                         name: 'Default Location',
+                         type: 'Location',
+                         ignore_types: ['ProvisioningTemplate', 'Hostgroup'],
+                         label: 'Default_Location',
+                         title: 'Default Location'
+                       ),
+                       :organization => Organisation.new(
```

Comment:
The tests are failing on typo, should be `Organization`. Have you tried actually running the tests? Katello's docs describe how to run single test https://www.theforeman.org/plugins/katello/developers.html#testing

Reply:
The last tests I did not found where it fails, I have used `rake test:katello`, and stuff that needed to fail did not, for example I placed location as `nil` and accessed `label`, but it did not failed.
I don't understand why yet.